pyroomacoustics/room.py

```diff
@@ -2189,15 +2154,64 @@ def extrude(self, height):
 
         Room.extrude(self, np.array([0.0, 0.0, height]))
 
-        # update the shoebox dim
-        self.shoebox_dim = np.append(self.shoebox_dim, height)
-
     def get_volume(self):
 
-        """
-        Computes the volume of a room
-        :param room: the room object
-        :return: the volume in cubic unit
-        """
-
         return np.prod(self.shoebox_dim)
+
+
+class AnechoicRoom(ShoeBox):
+    """ 
+    This class provides an API for creating an Anechoic "room" in 2D or 3D.
+
+    Parameters
+    ----------
+    dim: int
+        Dimension of the room (2 or 3).
+    kwargs:
+        All other kwargs are passed on to the :py:obj:`pyroomacoustics.room.Room` initializer.
+    """
+
+    def __init__(self, dim, **kwargs):
```

Comment:
Ok, your call! Since it is also the convention in the rest of the package, I guess it makes sense to follow the same convention, for consistency purposes.

Reply:
I rewrote this and agree now that this makes the code more readable!